elixir/apps/web/lib/web/live/sign_up.ex

```diff
@@ -37,6 +37,15 @@ defmodule Web.SignUp do
         actor: %{type: :account_admin_user}
       })
 
+    socket =
+      assign(socket,
+        form: to_form(changeset),
+        account: nil,
+        sign_up_enabled: Config.sign_up_enabled?()
```

Comment:
Just a convention to name boolean variables:
```suggestion
        sign_up_enabled?: Config.sign_up_enabled?()
```